main/src/ui/contact_details/muc_config_form_provider.vala

```diff
@@ -80,7 +80,7 @@ public class MucConfigFormProvider : Plugins.ContactDetailsProvider, Object {
                     break;
                 case "muc#roomconfig_historylength":
                     label = _("Message history");
-                    desc = _("Maximum amount of backlog issued by the room");
```

Comment:
"Backlog" might not be clear to people, backlog of what? What does "issued" mean?